circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
```

Comment:
> I asked @wilzbach to stick with one, because the Terms Of Service only offer 4 instances for open source projects. 

Did you miss this comment?

> I just found the following in the settings at the [org page](https://circleci.com/gh/organizations/dlang/settings)

>> Additionally, projects that are __public on GitHub__ will build with 3 extra containers -- our gift to free and open source software.

>and then this in their [FAQ](https://circleci.com/gh/organizations/dlang/settings#containers)

>>> What if I am building open-source?

>> We offer a total of four free linux containers ($2400 annual value) for open-source projects. __Simply keeping your project public__ will enable this for you!

https://github.com/dlang/dmd/pull/6006#issuecomment-237619185